docs/manual/java/guide/cluster/PersistentEntity.md

```diff
@@ -20,19 +20,21 @@ When an entity is started it replays the stored events to restore the current st
 
 ## Choosing a database
 
-Lagom supports the following databases:
+Lagom is compatible with the following databases:
 
-* [Cassandra](https://cassandra.apache.org/)
-* [PostgreSQL](https://www.postgresql.org/)
-* [MySQL](https://www.mysql.com/)
-* [Oracle](https://www.oracle.com/database/index.html)
-* [H2](https://www.h2database.com/)
+- [Cassandra](https://cassandra.apache.org/)
+- [PostgreSQL](https://www.postgresql.org/)
+- [MySQL](https://www.mysql.com/)
+- [Oracle](https://www.oracle.com/database/index.html)
+- [H2](https://www.h2database.com/)
+- [Microsoft SQL Server](https://www.microsoft.com/en-us/sql-server/)
```

Comment:
I think it's a good idea to at least run a manual smoke test with MS SQL Server before declaring support for it. We've seen problems with nearly every other RDBMS when we tested them.

Reply:
I've removed the reference to SQL Server to unblock this PR. We can add it later.